File: LV2/2.0/aboveaverage/aboveaverage.py
n = int(input())

for _ in range(n):
    m, *score = [int(i) for i in input().split(' ')]
    mean = sum(score)/float(m)
    remain = len([i for i in score if i > mean])
    print(f"{100*remain/float(m):.3f}%")

File: LV2/2.0/aboveaverage/aboveaverage.rb
n = gets.to_i

n.times do
  m, *score = gets.split.map(&:to_f)
  mean = score.sum / m
  remain = score.select { |i| i > mean }
  puts "#{format('%.3f', 100 * (remain.size / m))}%"
end
